src/Workspaces/Remote/ServiceHub/Services/LanguageServer.cs

```diff
@@ -48,7 +50,8 @@ public Task<InitializeResult> Initialize(JToken input, CancellationToken cancell
             {
                 Capabilities = new VSServerCapabilities()
                 {
-                    WorkspaceStreamingSymbolProvider = true
+                    DisableGoToWorkspaceSymbols = true,
```

Comment:
Is there documentation on these capabilities? I couldn't find them in the spec at https://microsoft.github.io/language-server-protocol/specifications/specification-current/

Reply:
DisableGoToWorkspaceSymbols and WorkspaceStreamingSymbolProvider are both custom VS capabilities.  DisableGoToWorkspaceSymbols disables using LSP for goto (cntrl + ,).  WorkspaceStreamingSymbolProvider is a now deprecated capability that indicated that the client supported streaming workspace symbols.  Now that is baked into the actual workspace/symbols request.

Reply:
To expand on this a bit.. ideally, VS uses only the public protocol and capabilities, however, we are dealing with:

- GoTo, which has an in-proc provider model _and_ support for OOP extensibility via LSP
- The need for streaming support to enable Ctrl+Q, which wasn't in the spec until post 16.3.

StreamingWorkspaceSymbols was a temporary measure until streaming was standardized, and DisableGoToWorkspaceSymbols is temporary until all partner teams switch to LSP for GoTo entirely instead of using both in-proc and OOP symbol providers.

3rd parties shouldn't have this problem as long as they choose one mechanism or the other.